<launch>
    <arg name="camera_name" default="sony_cam2"/>
    <arg name="tag_family" default="tag36h11"/>
    <arg name="tag_size" default="0.020"/>
    <arg name="visualize" default="True"/>

    <!-- Node definition with parameters in private namespace -->
    <node name="$(arg camera_name)_detect_april" pkg="tesol_detect" type="april_marker.py" output="screen">
        <param name="camera_name" value="$(arg camera_name)" />
        <param name="tag_family" value="$(arg tag_family)" />
        <param name="tag_size" value="$(arg tag_size)" />
        <param name="visualize" value="$(arg visualize)" />
    </node>
</launch>
